clients/ts/webpack.config.base.js

```diff
@@ -17,6 +17,10 @@ module.exports = function (modulePath, browserBaseName, options) {
             process: false,
             Buffer: false,
         },
+        resolveLoader: {
+            // Special resolution rules for loaders (which are in the 'common' directory)
+            modules: [ path.resolve(__dirname, "common", "node_modules") ],
```

Comment:
This is the main change for the PR?

Reply:
Yeah, the restructuring and then this change was needed to allow the WebPack loaders to resolve out of the other directory.